src/status_im/contexts/communities/actions/invite_contacts/view.cljs

```diff
@@ -0,0 +1,130 @@
+(ns status-im.contexts.communities.actions.invite-contacts.view
+  (:require
+    [quo.core :as quo]
+    [quo.foundations.resources :as resources]
+    [quo.theme]
+    [react-native.core :as rn]
+    [react-native.gesture :as gesture]
+    [react-native.share :as share]
+    [status-im.common.contact-list-item.view :as contact-list-item]
+    [status-im.common.contact-list.view :as contact-list]
+    [status-im.contexts.communities.actions.invite-contacts.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- no-contacts-view
+  [{:keys [theme id]}]
+  (let [customization-color             (rf/sub [:profile/customization-color])
+        {:keys [universal-profile-url]} (rf/sub [:profile/profile])
+        on-press-share-community        (rn/use-callback
+                                         #(rf/dispatch [:communities/share-community-url-with-data id]))
+        on-press-share-profile          (rn/use-callback #(share/open {:url universal-profile-url}))]
```

Comment:
Since you're using `rn/use-callback` only on the mount phase, there's no point in using `use-callback`.

But not going too much into that, I think here we should simply remove the form-2 component, move the subs to the renderer and update the `use-callback` call to have a dependency on `universal-profile-url`.